features/admin.users.v1/components/user-profile.tsx

```diff
@@ -230,6 +231,12 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
         featureConfig?.users?.scopes?.update
     );
 
+    const entitlementConfig: FeatureAccessConfigInterface = useSelector(
+        (state: AppState) => state?.config?.ui?.features?.entitlement);
+    const hasRoleV3UpdateScopes: boolean = useRequiredScopes(entitlementConfig?.scopes?.update);
```

Comment:
let's follow the pattern used in other places for variables that hold the scope check result.

```suggestion
    const hasRoleV3UpdatePermissions: boolean = useRequiredScopes(entitlementConfig?.scopes?.update);
```

Reply:
Fixed with: https://github.com/wso2/identity-apps/pull/8184/commits/364f88e7a85aa677d870962f791af4ab219be515